extensibility/metrics-reporter/server/main.go

```diff
@@ -26,18 +26,25 @@ package main
 
 import (
 	"log"
+	"time"
 
+	"github.com/uber-go/tally/v4"
 	"go.temporal.io/server/temporal"
+	servermetrics 	"go.temporal.io/server/common/metrics"
 
 	"github.com/temporalio/service-samples/metrics-reporter"
 )
 
 func main() {
+	options := tally.ScopeOptions{Reporter: metrics.NewReporter()}
+	scope, _ := tally.NewRootScope(options, time.Second)
+	reporter := servermetrics.NewTallyReporter(scope, &servermetrics.ClientConfig{})
```

Comment:
could you switch to use OTel reporter? I think we want to move away from tally in general.

Reply:
Keeping this for people who want to migrate. Otherwise, there's another PR using otel.
I can close this one in favor of https://github.com/temporalio/samples-server/pull/25